### PR TITLE
add javascript example

### PR DESCRIPTION
**Pull request by ziquanmiao**
leverage nodeforge to make a client facing javascript example

### CODE REVIEW

**Review by RodionovDmitry — APPROVED** [review]

File: JavaScriptSample.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <script src="https://cdn.jsdelivr.net/npm/node-forge@0.7.0/dist/forge.min.js"></script>
```
Comment (roktTG) [inline JavaScriptSample.html]: Could you try to re-implement the example using [Subtle Crypto](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto)? The implementation is native to the browser and will not require loading additional, quite chunky, library which some partners may be iffy about.
You can see in [the supported algorithms](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto#supported_algorithms) both 'RSA-OAEP' and 'sha256' are supported.

Comment (ziquanmiao) [reply JavaScriptSample.html]: looking into this!

Comment (ziquanmiao) [reply JavaScriptSample.html]: added support for subtle crypto

File: JavaScriptSample-subtlecrypto.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div></div>
+
+    <script>
+      (async () => {
+        const publicKey = "xxx"; //add the public encryption key here
+        const TO_ENCRYPT = "hotdog_is_a_s@ndwich.com"; // this is what we are encrypting
```
Comment (Ethaywien) [inline JavaScriptSample-subtlecrypto.html]: why is `publicKey` camel case and `TO_ENCRYPT` upper snake case?

Comment (ziquanmiao) [reply JavaScriptSample-subtlecrypto.html]: 🤦 yes will change casing to be uniform

**Review by Ethaywien — APPROVED** [review]

**Pull request merged**